symplyphysics/laws/astronomy/luminosity_of_sun_in_past_from_luminosity_in_present.py

```diff
@@ -0,0 +1,46 @@
+from sympy import (Eq, solve, S,)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    convert_to,
+    dimensionless,
+)
+
+# Description
+## The luminosity of the Sun in the past is related to the luminosity of the Sun in the present. Luminosity is
+## indicated in units of solar luminosity. One unit is equal to the luminosity of the Sun at a given time.
+## The luminosity in the past is the luminosity of the sun with a lower age than at the current moment. The luminosity
+## in the future is the luminosity of the sun with a greater age than at the current moment.
```

Comment:
It's obvious. I meant that you should explain, why there are two different formulas to calculate luminosity in the past and in the future.

Reply:
> It's obvious. I meant that you should explain, why there are two different formulas to calculate luminosity in the past and in the future.

I don't know. This was not mentioned in the source.

Reply:
It is because it is non linear and requires different approximations for past and future. You can also add link to a picture that shows this dependency.